programs/survival/cam.sc

```diff
@@ -5,7 +5,11 @@
 // places spectating players without config in a safe places
 // and adds timeout for the player so they cannot switch instantly
 // change setting below to change that behaviour (0 to disable timeout)
-global_survival_timeout = 3;
+global_survival_timeout = 0;
+
+// optionally, you can restrict a player from flying too far from their original 
+// position (null or any non positive number to disable)
+global_max_flight_range = 10;
```

Comment:
Tbh most people are fine with /cam allowing you to move as far as you want, so maybe keep this off by default?

Reply:
absolutelyu, again. forgot to change defaults.